orderbook/src/database/fees.rs

```diff
@@ -11,14 +11,18 @@ use ethcontract::{H160, U256};
 impl MinFeeStoring for Database {
     async fn save_fee_measurement(
         &self,
-        token: H160,
+        sell_token: H160,
+        buy_token: Option<H160>,
+        sell_amount: Option<U256>,
         expiry: DateTime<Utc>,
         min_fee: U256,
     ) -> Result<()> {
         const QUERY: &str =
-            "INSERT INTO min_fee_measurements (token, expiration_timestamp, min_fee) VALUES ($1, $2, $3);";
+            "INSERT INTO min_fee_measurements (sell_token, buy_token, sell_amount, expiration_timestamp, min_fee) VALUES ($1, $2, $3, $4, $5);";
         sqlx::query(QUERY)
-            .bind(token.as_bytes())
+            .bind(sell_token.as_bytes())
+            .bind(buy_token.map(|t| t.as_bytes().to_owned()))
```

Comment:
this is how we can avoid converting to vector
```suggestion
            .bind(buy_token.as_ref().map(|t| t.as_bytes()))
```